cmd/apichecker/updates.json

```diff
@@ -275,23 +275,6 @@
    },
    "Disabled": false
   },
-  {
-   "Name": "LakeBTC",
-   "CheckType": "HTML String Check",
-   "Data": {
-    "HTMLData": {
-     "TokenData": "div",
-     "Key": "class",
-     "Val": "flash-message",
-     "TokenDataEnd": "h2",
-     "TextTokenData": "h1",
-     "RegExp": "APIv\\d{1}",
-     "CheckString": "de2491b95ef1f6ea334247b13f0f14f6816fb5961cc63acc0542b07fc0336dd8",
-     "Path": "https://www.lakebtc.com/s/api_v2"
-    }
-   },
-   "Disabled": false
-  },
```

Comment:
There is a committed file for the api checker `backup.json` that still mentions LakeBTC if we're going for wide-scale destruction

Reply:
nuke.exe fully loaded, similar to crypto markets ATM 👀 😆